tdrs-backend/tdpservice/email/tasks.py

```diff
@@ -69,3 +73,86 @@ def email_admin_num_access_requests():
                                    subject,
                                    email_context,
                                    )
+
+
+@shared_task
+def send_data_submission_reminder(due_date, reporting_period, fiscal_quarter):
+    """Send all Data Analysts a reminder to submit if they have not already."""
+    def get_fiscal_year(calendar_year, fiscal_quarter):
+        return calendar_year - 1 if fiscal_quarter == 'Q1' else calendar_year
+
+    now = datetime.now()
+    fiscal_year = get_fiscal_year(now.year, fiscal_quarter)
+
+    all_locations = STT.objects.all()
+
+    #
+    reminder_locations = []
+    year_quarter_files = DataFile.objects.all().filter(
+        year=fiscal_year,
+        quarter=fiscal_quarter,
+    )
+
+    for loc in all_locations:
+        submitted_sections = year_quarter_files.filter(stt=loc).values_list('section', flat=True).distinct()
+        required_sections = loc.filenames.keys()
+
+        submitted_all_sections = True
+        for s in required_sections:
+            if s not in submitted_sections:
```

Comment:
I might be pushing this too far and or making it less readable. But, we could avoid the `if` statement here by initially setting `submitted_all_sections = True` (which you have done) and then replacing the `if` statement with `submitted_all_sections &= s not in submitted_sections`. The bitwise `&=` will keep that boolean set to `False` no matter what if `s not in submitted_sections` ever evaluates to `False`.